src/components/Footer.js

```diff
@@ -1,7 +1,3 @@
 export default function Footer() {
-	return (
-		<footer>
-			<div>© 2022 by Me</div>
-		</footer>
-	);
+	return <footer>{/* <div>© 2022 by Me</div> */}</footer>;
 }
```

Comment:
If you do not use it, you should delete it 🧹

Reply:
deleted